src/routes/safe/store/actions/createTransaction.js

```diff
@@ -55,6 +56,9 @@ const createTransaction = ({
     dispatch(push(`${SAFELIST_ADDRESS}/${safeAddress}/transactions`))
   }
 
+  const ready = await onboardUser()
```

Comment:
This feels weird (from a non-react developer point): this is part of our store/ logic layer and we include a UI component :/

Reply:
It's not an UI component, it's just placed there. Perhaps moving this function to a separate file is a good idea.